lib/foreman_maintain/upgrade_runner.rb

```diff
@@ -209,7 +209,7 @@ def confirm_scenario(scenario)
         response = reporter.ask_decision(<<-MESSAGE.strip_heredoc.strip)
             The pre-upgrade checks indicate that the system is ready for upgrade.
             It's recommended to perform a backup at this stage.
-            Confirm to continue with the the modification part of the upgrade
+            Confirm to continue with the modification part of the upgrade
```

Comment:
One of test is failing for this change in [upgrade_runner_test.rb file.](test/lib/upgrade_runner_test.rb) Please fix failing test.